Pilot:
    dt: 0.01
    reference_frame: "odom/unfiltered" # "odom/unfiltered", "estimated_odom"
    controller_type: "SMC" # "PID" , "SMC"
Control:
    PID:
        Kp: [1.5, 1.5, 1.2]
        Kd: [0.8, 0.8, 0.9]
        Ki: [0.03, 0.03, 0.02]
        windup: [1.0, 1.0, 1.0]
        saturation: [0.8, 0.8, 0.4]
    SMC:
        # Model Based
        # K: [4.5, 5, 1.2]
        # alpha: [3.5, 2.2, 2.0]
        # A: [0.4, 0.4, 0.15]
        # saturation: [100, 100, 20]

        K: [0.5, 0.5, 0.5]
        alpha: [3.5, 2.2, 2.0]
        A: [0.4, 0.4, 0.15]
        saturation: [100, 100, 20]

        # Model Free
        # K: [90.0, 120.0, 20.0]
        # alpha: [6.2, 6.2, 8.0]
        # A: [0.4, 0.4, 0.15]
        # saturation: [0.8, 0.8, 0.4]

        useModel: True # Allows model-based control.
